src/analyze.test.ts

```diff
@@ -336,9 +374,28 @@ Object {
           "isTemplate": false,
           "name": undefined,
           "of": "./Button.stories",
+          "tags": Array [],
           "title": undefined,
         }
       `);
     });
   });
+
+  describe('markdown/html', () => {
+    it('title', () => {
+      const input = dedent`
+      <title>foobar</title>
```

Comment:
I'd love to do a call about to discuss this before merging. Not sure I agree with this direction for the tool, but haven't really thought it through.

Reply:
Sure thing